repo/packages/O/openldap-admin/0/marathon.json.mustache

```diff
@@ -0,0 +1,43 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{openldap-admin.cpus}},
+    "mem": {{openldap-admin.mem}},
+    "instances": 1,
+    "env": {
+        "PHPLDAPADMIN_LDAP_HOSTS": "{{openldap-admin.ldap_hosts}}",
+        "PHPLDAPADMIN_HTTPS": "{{openldap-admin.https}}",
+        "LDAP_ADMIN_PASSWORD": "{{openldap.admin_password}}",
+        "PHPLDAPADMIN_LDAP_CLIENT_TLS": "{{openldap-admin.client_tls}}"
+    },
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.openldap-admin-docker}}",
+            "forcePullImage": true,
+            "network": "BRIDGE",
+            "portMappings": [
+            {
+                "containerPort": 443,
+                "hostPort": 0,
+                "protocol": "tcp"
+            }
+            ]
+        }
+    },
+    "acceptedResourceRoles": ["slave_public"],
```

Comment:
Sure? This should be available via. admin router and that'll be authenticated at least.